plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -201,6 +201,7 @@
 import static java.lang.String.format;
```

Comment:
Can you please add link to postgresql documentation that proves what is a canonical form of an identifier in Postgresql?